libs/oef-base/include/oef-base/utils/Uri.hpp

```diff
@@ -59,12 +59,12 @@ class Uri
                    "\" port=", port, " path=\"", path, "\"");
   }
 
-  inline std::string GetSocketAddress() const
```

Comment:
Why not inlining these 2 methods?

Reply:
Member functions defined inside the class definition are implicitly `inline`.